src/shell/Session.ts

```diff
@@ -22,7 +22,7 @@ export class Session extends EmitterWithUniqueID {
     history = History;
     historicalPresentDirectoriesStack = new OrderedSet<string>();
 
-    constructor(private application: ApplicationComponent, private _dimensions: Dimensions) {
+    constructor(private application: ApplicationComponent, private _dimensions: Dimensions, public position: Positions) {
```

Comment:
where do you think the best place to store that? in `SessionComponent`, correct?